.github/workflows/canary-integration-test.yml

```diff
@@ -993,7 +993,12 @@ jobs:
       run: |
         kubectl -n rook-ceph get cephfilesystem myfs -o yaml
         # the check is not super ideal since 'mirroring_failed' is only displayed when there is a failure but not when it's working...
-        timeout 60 sh -c 'while [ "$(kubectl exec -n rook-ceph deploy/rook-ceph-tools -t -- ceph fs snapshot mirror daemon status myfs|jq -r '.[0].filesystems[0]'|grep -c "mirroring_failed")" -eq 1 ]; do echo "waiting for filesystem to be mirrored" && sleep 1; done'
+        # fs-name needed for Ceph v16.2.6 and earlier
+        if kubectl exec -n rook-ceph deploy/rook-ceph-tools -t -- ceph fs snapshot mirror daemon status myfs &> /dev/null; then
```

Comment:
I'm wondering if this is working as intended... How do we differentiate between the command fails because the mirroring is not ready and the command does not exist?

This is what I get in 16.2.7:

```
[leseb@tarox~/go/src/github.com/rook/rook][master *] kubectl exec -n rook-ceph deploy/rook-ceph-tools -t -- ceph fs snapshot mirror daemon status
Error ENOENT: filesystem None does not exist
command terminated with exit code 2
[leseb@tarox~/go/src/github.com/rook/rook][master *] 
[leseb@tarox~/go/src/github.com/rook/rook][master *] 
[leseb@tarox~/go/src/github.com/rook/rook][master *] kubectl exec -n rook-ceph deploy/rook-ceph-tools -t -- ceph fs snapshot mirror daemon status myfs
Error EINVAL: filesystem myfs is not mirrored
command terminated with exit code 22
```

I'm not sure how the next version behaves, have you tried `latest-pacific-devel` locally?

Reply:
I didn't expect this type of output from 16.2.7,
This cmd `ceph fs snapshot mirror daemon status` shall have return mirroring not ready if its not ready,
and `ceph fs snapshot mirror daemon status myfs` have return an Invalid command.


As I tested and quoted [above](https://github.com/rook/rook/pull/8776#issuecomment-951911444).
I have tested it using MultiClusterMirroringCI only.